### PR TITLE
Include rth

### PR DESCRIPTION
**Pull request by ulbricht-inr**
Hi, 

could you please include this new Raspberry-HAT into your repository.
local publish works for me.

Thank you very much.
Greetings. 
Marian

### CODE REVIEW

File: draft/overlay/RealTimeHAT.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+<!--
+---
+name: RealTimeHAT
+class: board
+type: multi
+formfactor: HAT
+manufacturer: InnoRoute GmbH
+description: A sblabalbal board that can be used easily to show status of the Pi
```
Comment (lurch) [inline draft/overlay/RealTimeHAT.md:8]: Is that a typo, or is `sblabalbal` a super-obscure word I've never heard of? :laughing:

File: draft/overlay/RealTimeHAT.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    name: IO Expender
+    device: I2C1 - IO Expander
+-->
+# RealTimeHAT
+
+The RealTimeHAT extends the Gigabit-Ethernet interface of the Raspberry Pi by professional networking functions.
+Typical applications are **industrial real-time communication** and **network monitoring**.
+
+The HAT provides **3 Gigabit-Ethernet ports**, one of which connecting to the Raspberry Pi's Ethernet port.
+Connection to the Raspberry Pi is realized via the GPIO connector and via a (short) Ethernet cable. Interface SPI0 is for main configuration, using either CE0# or CE1#. Beyond the standard interfaces, the HAT uses 4 dedicated pins
```
Comment (lurch) [inline draft/overlay/RealTimeHAT.md:73]: I guess it's not really clear what "Beyond the standard interfaces" means? :shrug:

File: draft/overlay/RealTimeHAT.md (hunk 1 above)
Comment (ulbricht-inr) [reply draft/overlay/RealTimeHAT.md:8]: oops :D

File: draft/overlay/RealTimeHAT.md (hunk 2 above)
Comment (ulbricht-inr) [reply draft/overlay/RealTimeHAT.md:73]: right, this sentence is not necessary users can simply read the pinout

**Comment by Gadgetoid** [comment]
Will get to it ASAP. Currently frantically packing to move house.

**Comment by lurch** [comment]
> Currently frantically packing to move house.

Oooh, good luck! :crossed_fingers: :house:

**Comment by ulbricht-inr** [comment]
Thanks :)

**Comment by ulbricht-inr** [comment]
Hi @Gadgetoid i know its currently X-mas holiday time :D 
But cut you please push the button to have the current state of the Repo (including the Realtime-Hat) online at pinout.xyz. I thought this is scripted but there was no change in the last weeks.
Thank you, and a good Start into 2021 for you :)

**Pull request merged**